### PR TITLE
Add support to navigation

### PR DESCRIPTION
**Pull request by Lukewh**
## Done

Added `Support` to navigation

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check the content matches the [copy doc (second part)](https://docs.google.com/document/d/1cLgHBSlRM-yOGifOz-vexDAq8XaW-TBzmGMtJMYbRZU/edit?ts=5b473e3a#)
- Check all the links go to the right places

### CODE REVIEW

**Comment by frankiegranato** [comment]
That looks excellent Luke! And so fast. 
Can you move 'Support' so it comes after 'Enterprise' please? So order would be Enterprise, Support, Developer, Community.

**Comment by anasereijo** [comment]
Thanks @Lukewh :)

can you please make the "* Minimums apply" h6?

**Comment by anasereijo** [comment]
@Lukewh sorry forgot to mention to make "* Minimums apply" 14px

**Comment by anthonydillon** [comment]
## QA
- on small screens you hide the prices but don't hide the note about "* Minimums apply"
- the copy doc has now links on the list items

File: templates/templates/_navigation-support-h.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            <li class="p-list__item">Extended Security Maintenance</li>
+            <li class="p-list__item">Livepatch: automatic kernel security hotfixes</li>
+            <li class="p-list__item">Landscape: the systems management tool</li>
+            <li class="p-list__item">Knowledge Base</li>
+            <li class="p-list__item">Support: phone and web</li>
+          </ul>
+        </div>
+      </div>
+      <div class="row">
+        <p class="p-heading--six-original">* Minimums apply</p>
```
Comment (anthonydillon) [inline templates/templates/_navigation-support-h.html]: Could you just use `<small><i>* Minimums apply<I><small>` save us another snowflake

Comment (Lukewh) [reply templates/templates/_navigation-support-h.html]: Done

**Comment by Lukewh** [comment]
@anthonydillon I didn't realise that entire block was in a 'hide' so I've added an extra line to show the price on small screens. I tried wrapping the heading and the `<small>` in a `<div>` with the classes on, but it pushed the items next to each other - I've added a comment to the code.

I'm not sure what you mean - I've added the links to the items that have them in the copy doc. I've built the second version (the table one) from the doc.

**Comment by anasereijo** [comment]
The navigations links on 320px screens isn't looking ok - there's not enough space for so many links.

screen shot 2018-07-13 at 09 13 02

My suggestions is to make it possible to scroll (horizontally) like google does:

img_8861

**Comment by frankiegranato** [comment]
This is not approved by Mark. We should re-think this.

**Comment by anthonydillon** [comment]
@Lukewh sorry, think we should kill this branch at this stage.

**Comment by Lukewh** [comment]
@anthonydillon go ahead, no worries.

**Pull request closed without merging**